mne/report.py

```diff
@@ -1556,6 +1574,50 @@ def _render_forward(self, fwd_fname):
                                         repr=repr_fwd)
         return html
 
+    def _render_sensitivity_map(self, fwd_fname, subjects_dir):
+        """Render Sensitivity plots.
+        """
+        caption = u'Sensitivity : %s' % fwd_fname
+        div_klass = 'sensitivity'
+        img_klass = 'Sensitivity'
+        show = True
+
+        fwd = read_forward_solution(fwd_fname, surf_ori=True)
+        mag_map = sensitivity_map(fwd, ch_type='grad', mode='fixed')
+        mag_map.plot(time_label='Gradiometer sensitivity',
+                 subjects_dir=subjects_dir,
```

Comment:
you are still having pep8 problems ... can you make sure you have it integrated in your editor? Otherwise people will complain before looking at anything else in your PR